src/pages/registration/SupportModalContent.tsx

```diff
@@ -1,17 +1,19 @@
-import { useQuery } from '@tanstack/react-query';
-import { useTranslation } from 'react-i18next';
+import { useMutation, useQuery } from '@tanstack/react-query';
+import { Trans, useTranslation } from 'react-i18next';
 import { useDispatch } from 'react-redux';
-import { fetchRegistrationTickets } from '../../api/registrationApi';
+import { addTicketMessage, createTicket, fetchRegistrationTickets } from '../../api/registrationApi';
 import { setNotification } from '../../redux/notificationSlice';
 import { Registration } from '../../types/registration.types';
-import { ActionPanelContent } from '../public_registration/ActionPanelContent';
+import { Divider, Grid, Link as MuiLink, Skeleton, Typography } from '@mui/material';
+import { MessageForm } from '../../components/MessageForm';
+import { OpenInNew } from '@mui/icons-material';
```

Comment:
Hør med @anetteOlli eller @joachimjorgensen om hvordan sette opp sortering av imports i IntelliJ (om det er det du bruker?)
Dette for å unngå at PRs består av store diffs som bare er flytting av imports som er uinteressant, og fjerner fokus på det som faktisk er endret. Har en notis i bakhodet om å få lagt inn opinionated tooling på dette så det ikke blir editor-avhenging. Men er ikke sånn at jeg kommer til å rekke å sette det opp uansett da 😅